source/features/avoid-accidental-submissions.tsx

```diff
@@ -14,27 +15,37 @@ function onKeyDown(event: delegate.Event<KeyboardEvent, HTMLInputElement>): void
 		|| event.ctrlKey
 		|| event.metaKey
 		|| event.isComposing // #4323
-		|| select.exists('.suggester', event.delegateTarget.form!) // GitHub’s autocomplete dropdown
+		|| select.exists([
+			'.suggester', // GitHub’s autocomplete dropdown
+			'.toRemove',
+		],
+		event.delegateTarget.form!)
 	) {
 		return;
 	}
 
+	const message = <p className="toRemove">A submission via <kbd>enter</kbd> has been prevented. You press <kbd>enter</kbd> again or use <kbd>ctrl</kbd>-<kbd>enter</kbd> next time</p>;
```

Comment:
Use classes that start with “rgh-“ and are lowercase so we always know they’re ours.

Reply:
Also this likely needs `.mt-1`